### PR TITLE
Fixed scaling on the parallax background, and also set the import set…

### PR DESCRIPTION
**Pull request by JustinBezzant**
fixed parallax background and updated sprite import sizes

### CODE REVIEW

File: Assets/Scripts/Parallax.cs [hunk 1]
```diff
@@ -32,15 +30,7 @@ void Update()
...
-            if (GameCamera.transform.position.y > GenerationHeight[i])
-            {
-                Sprites[i].SetActive(true);
-            }
-            else
-            {
-                Sprites[i].SetActive(false);
-            }
-            Sprites[i].transform.position = new Vector3(XOffset[i], YMovement(Sprites[i], YOffset[i], Mag[i]), 0);
+            Sprites[i].transform.position = new Vector3(0, YMovement(Sprites[i], YOffset[i], Mag[i]), 0);
```
Comment (taeuscherpferd) [inline Assets/Scripts/Parallax.cs]: When this is a list, the syntax will be Sprites.at(i)

File: Assets/Scripts/Parallax.cs [hunk 2]
```diff
@@ -9,8 +9,6 @@ public class Parallax : MonoBehaviour
 
     public float[] Mag;
     public float[] YOffset;
```
Comment (taeuscherpferd) [inline Assets/Scripts/Parallax.cs]: If you could change these to lists, everything else looks great 👍 nice job!

File: Assets/Scripts/Parallax.cs (hunk 1 above)
Comment (JustinBezzant) [reply Assets/Scripts/Parallax.cs]: I'm going to be removing the list and array parts completely

Comment (JustinBezzant) [reply Assets/Scripts/Parallax.cs]: fixed the list's/array's issue

**Pull request merged**